pkg/catalog/endpoint_test.go

```diff
@@ -21,4 +21,17 @@ var _ = Describe("Test catalog functions", func() {
 			Expect(actual).To(Equal(expected))
 		})
 	})
+
+	Context("Testing GetResolvableServiceEndpoints()", func() {
+		It("returns the endpoint for the service", func() {
+			actual, err := mc.GetResolvableServiceEndpoints(tests.BookstoreService)
```

Comment:
Could you add test for the case where the service does not have cluster IP set but has endpoints backing it?

Reply:
I have few on stash. Working on it.